### DISCUSSION TITLE
Is there a guide for how to setup backups on Graphene OS?

### DISCUSSION THREAD

**Discussion by terry9873** [seedvault-app/seedvault#406] [General]
I am sorry if this shouldn't belong here. I have never really understood github, very few things have instructions and that's because most people on the site have code-level knowledge. I don't! I am told this app is used for backups on GrapheneOS, but I can't find any instructions. If I have overlooked it, could someone please drop me a link. If not, could someone advise what a normal user has to do to get backups running on a device, I want to backup to a USB stick (which I am told works on Graphene via Seedvault)
thanks

**Comment by macbugs** [comment]
> **I have never really understood github**

Github should deploy a less complicated web interface for the general public that only needs to ask a question or report a bug -- but you did nothing wrong by posting in the discussion forum.  (Many github projects have not even enabled this feature yet.)

> **I am told this app is used for backups on GrapheneOS, but I can't find any instructions.  If I have overlooked it, could someone please drop me a link.**

Did you have any usage question which is not answered by this [introductory video](https://calyxinstitute.org/projects/seedvault-encrypted-backup-for-android)?

If your device is on the [compatibility list](https://doc.e.foundation/devices), you might want to consider switching from Graphene to /e/ OS, since it has a robust [support forum](https://community.e.foundation/) and live chat service.  I certainly understand if you want local backups because you dont trust Google -- but if you would prefer automatic online backups, the /e/ team is aiming to replicate [features of the Google ecosystem](https://e.foundation/ecloud/) in a more secure fashion.

**Comment by terry9873** [comment]
Hi, thanks for the reply. Yes I am always nervous on github, feel out of my depth and also feel like I am intruding quite often as I rarely see anyone on here without significant knowledge of code etc!  The trouble comes when the only info on an app I am interested in, is located on Github!! 

That video is helpful, so I can basically just plug in a USB stick and do as it shows. That's neat! I have never heard of /e/, but I want no cloud services whatsoever so I don't see it being useful to me. Great for those already hooked on sharing their files around their devices etc though. thanks again

**Comment by terry9873** [comment]
Sorry I meant to ask something else... I never really understand just how much of a backup, a backup actually is! (Mac user, Timemachine, CarbonCopyCloner, etc!). 
So if i do as the video shows, would that USB stick be able to take a new phone to the exact state my phone is in? I know backups don't include Signal (which is fine), but hopefully things like email  imap settings, my music and audiobooks, newpipe (and settings) would go onto a device if i restored from the backup? 
Thanks

**Comment by macbugs** [comment]
> **feel like I am intruding quite often**

A few project managers would prefer to see only bug reports & feature requests in the "issues" section (as opposed to usage questions), but if that is the case then they need to enable discussions.  Most will just tag your post with the "question" label if needed -- but if the app has its own support forum on some other site, you should probably post your question there first.  When posting on github, just make a reasonable effort to see if someone else has already posted about the same issue before you create a new post.

> **how much of a backup, a backup actually is! (Mac user, Timemachine, CarbonCopyCloner, etc!)**

That is entirely dependent on the app you are using for backup, so you would have to refer to the documentation for the app you want to use.

> would that USB stick be able to take a new phone to the exact state my phone is in

If SeedVault runs on the new phone, you should be able to migrate what you backed up.  If not, then you need a backup utility that runs on both platforms or a desktop tool which can make backups from your phone.  While SeedVault promises "comprehensive data backups", I dont see a list of what that includes.  The documentation seems rather poor and I was actually just looking for the SeedVault [official web site](https://seedvault.app/) (which is basically just a blank page at this point.)  If you dont get a satisfactory answer here, perhaps you could post in the [Graphene issue tracker](https://github.com/GrapheneOS/os-issue-tracker/issues).

**Comment by terry9873** [comment]
"That is entirely dependent on the app you are using for backup," - Sorry, I thought we were talking about SeedVault and I thought that was the app which handles the backups. I will look into it again. 

Yep i always look for support channels/forums etc. Many don't have them, but when they do I go there first. Believe me, for me, GitHub is a VERY last resort!

**Comment by macbugs** [comment]
> **I thought we were talking about SeedVault and I thought that was the app which handles the backups**

It is.  But you also mentioned _other_ backup apps (Timemachine, CarbonCopyCloner, etc!) in the context of wondering whether these apps will copy all user data.  So my answer to that was, _it depends on which backup tool you are using._  I cannot determine if SeedVault backups would include absolutely **all** user data (like SMS message history).  You did say _"the exact state my phone is in"_ -- but I cannot guarantee that SeedVault will replicate the "exact state" when I can find no published specifications.  If you are migrating your data to another phone that runs SeedVault, its up to you to confirm that everything was restored properly.  If SeedVault does not accomplish this to your satisfaction (or does not run on the new phone) you will have to use some other backup tool as I described.

**Comment by terry9873** [comment]
Ah. Yes sorry I shouldn't have confused the issue mentioning silly Apple apps :D. 
Yep gotcha, I am well used to checking if backups restored everything when it comes to desktop machines (for the aforementioned but neveragaintobementioned apps :P) and they often don't! CCC is the only one (as it's a clone) that does that. I will have a play with SeedVault, thanks for your help here.
